tools/ci_build/github/azure-pipelines/templates/c-api-linux-cpu.yml

```diff
@@ -33,14 +33,18 @@ parameters:
   type: boolean
   default: true
 
+- name: SkipComponentGovernanceDetection
+  type: boolean
+  default: false
+
 jobs:
 - job: Linux_C_API_Packaging_CPU_${{parameters.OnnxruntimeArch}}
 
   workspace:
     clean: all
   variables:
   - name: skipComponentGovernanceDetection
-    value: ${{ in('${{parameters.OnnxruntimeNodejsBindingArch}}', 'arm64', 'aarch64') }}
```

Comment:
Somehow this is True when arch is x64 and False when the arch is arm64

Reply:
What if you totally remove this condition, what would happen?

Reply:
If we remove the ${{ in('${{parameters.OnnxruntimeNodejsBindingArch}}', 'arm64', 'aarch64') }}, we need to it to either True or False. Value cannot be unset.

Reply:
I mean, what blocks us running component governance on ARM64? Can we fix that?

Reply:
I thought it wasn't supposed on arm64?

Reply:
Let me remove it and see